packages/lib/server-only/document/send-delete-email.ts

```diff
@@ -0,0 +1,71 @@
+import { createElement } from 'react';
+
+import { mailer } from '@documenso/email/mailer';
+import { render } from '@documenso/email/render';
+import { DocumentDeletedEmailTemplate } from '@documenso/email/templates/document-deleted';
+import { prisma } from '@documenso/prisma';
+import { DocumentStatus, SigningStatus } from '@documenso/prisma/client';
+
+export interface SendDocumentOptions {
+  documentId: number;
+  userId: number;
+}
+
+export const sendDeletedEmail = async ({ documentId, userId }: SendDocumentOptions) => {
+  const user = await prisma.user.findFirstOrThrow({
+    where: {
+      id: userId,
+    },
+  });
+  const document = await prisma.document.findUnique({
+    where: {
+      id: documentId,
+    },
+    include: {
+      documentData: true,
+      Recipient: true,
+    },
+  });
+
+  if (!document) {
+    throw new Error('Document not found');
+  }
+
+  if (document.status == DocumentStatus.DRAFT) {
+    throw new Error('Document is a draft state');
+  }
```

Comment:
Right I see so with the part I unresolved this would always throw an error for draft documents meaning you wouldn't be able to delete them.

Reply:
Now made redundant but could be kept for extra safety I suppose